main/resource-access.bs

```diff
@@ -5,28 +5,65 @@ Issue: Write introduction to the Authenticated Resource Access section.
 
 ## Hypertext Transfer Protocol ## {#http}
 
-Issue: Write Hypertext Transfer Protocol section.
-
-Draft:
-A Solid data pod MUST implement HTTP/1.1 Message Syntax and Routing [[!RFC7230]]
-
-Draft:
-A Solid data pod MUST implement HTTP/1.1 Semantics and Content [[!RFC7231]].
-
-Draft:
-A Solid data pod MUST implement HTTP/1.1 Authentication [[!RFC7235]].
-
-Draft:
-A Solid data pod SHOULD implement HTTP/1.1 Conditional Requests [[!RFC7232]].
-
-Draft:
-A Solid data pod SHOULD implement HTTP/1.1 Range Requests [[!RFC7233]].
-
-Draft:
-A Solid data pod SHOULD implement HTTP/1.1 Caching [[!RFC7234]].
-
-Draft:
-A Solid data pod MAY implement HTTP/2 [[!RFC7540]].
+### Background and Need ### {#http-need}
+<em>This section is non-normative.</em>
+
+Solid clients and servers need to exchange data securely over the Internet,
+and they do so using the Web protocol HTTP.
+This section describes in detail
+which parts of HTTP must be implemented by clients and servers.
+
+### Required server-side implementation ### {#http-server}
+
+A [=data pod=] MUST be an HTTP/1.1 server [[!RFC7230]][[!RFC7231]].
+It SHOULD additionally be an HTTP/2 server [[!RFC7540]]
+to improve performance,
+especially in cases where individual clients
+are expected to send high numbers of successive requests.
+
+A data pod SHOULD use the `https` URI scheme
```

Comment:
This formulation seems to be a departure from the ["it's not the 's'"](https://www.w3.org/DesignIssues/Security-NotTheS.html). I think it should emphasize what the s means, i.e. the use of TLS. I think the important part of the discussion is in the security section, which you have below.

Reply:
Wording is from RFC7230; do a grep for TLS there, and the most prominent mentions are in "https URI scheme" sections.

Happy to additionally mention TLS, just didn't want to be redundant. Do you have a proposal for phrasing?

Reply:
What I was trying to get at is that using the https by itself doesn't secure anything, the server admin could in principle have been duped into installing a middle-man's certificate. 

How about 

"A data pod SHOULD be secured with TLS, and therefore use the `https` URI scheme."